docs/CONTRIBUTING.md

```diff
@@ -7,85 +7,198 @@
-* Tokens are defined in JSON form, in palmetto-design-tokens.
-* When palmetto-design-tokens is built, it will generate utility classes, scss, and css variables based on these tokens.
+Our component library is based on the [Palmetto Design Tokens](https://github.com/palmetto/palmetto-design-tokens) design system.
+
+The design system provides us with a set of shared attributes used in the Palmetto Design System. These attributes include: colors, fonts, spacing, and more.
+* The library provides the project with utility classes, scss, css, JS, and JSON variables.
 * These classes and variables are imported globally in palmetto-components, and all styles are based on them.
-* Tokens should NEVER be component-specific, however component variables can be derived from tokens. E.G (with sass): `$form-control-height-medium: $size-spacing-medium;`
+* Tokens should NEVER be component-specific, however component variables can be derived from tokens. For e.g. (with sass): `$form-control-height-medium: $size-spacing-medium;`
+
+If there are additional attributes that you feel should be included in Palmetto Design Tokens, [raise an issue here](https://github.com/palmetto/palmetto-design-tokens/issues).
 
-## JS Style Guide
+## JS/JSX Style Guide
 
-We mostly follow the [AirBnB JS style guide](https://github.com/airbnb/javascript/tree/master/react) with few exceptions. Just follow linter rules as much as possible. Please use `// eslint-disable-line` very, very sparingly. If a disable is needed, please specify the rule you are disabling.
+We follow the [AirBnB JS style guide](https://github.com/airbnb/javascript/tree/master/react), with a few exceptions. Adhere to the linter rules as much as possible. In cases where you can't, use `// eslint-disable-line`, etc., very, very sparingly. If a disable is needed, specify the exact rule that you are disabling.
 
-E.G: `// eslint-disable-line no-console`
+For example: `// eslint-disable-line no-console`
 
-##  Sass/CSS Style Guide
+##  CSS/Sass Style Guide
 
-* TBD
+* We strictly adhere to the [Concentric CSS Model](https://rhodesmill.org/brandon/2011/concentric-css/) for organizing our CSS properties. If any properties are out of order when a PR is opened, the build will fail until the issues are corrected.
 
-## Writing Components
+## Building a Component
 
-Most if not all components in this library should be **controlled** components. Meaning that they hold no internal state. For an example of code structure for most components, review an existing component and adhere to the structure that you see. Some general rules:
+Most, if not all components in this library should be **controlled** components. Meaning that they hold no internal state. 
+
+A review of existing components will reveal that the library follows a set pattern for their organization; please review the components, and adhere to the structure that you see. Some general rules:
 
 * Follow order of PropType declarations (before/above component markup).
```

Comment:
@JCarmine I might actually specify that we're talking about typescript interfaces here not vanilla propTypes.